tracer/test/Datadog.Trace.Security.Unit.Tests/IAST/Tainted/vulnerability_schema.json

```diff
@@ -29,7 +29,8 @@
               "grpc.request.body",
```

Comment:
This file has been updated with the latest changes in https://github.com/DataDog/experimental/blob/main/teams/asm/vulnerability_schema/vulnerability_schema.json